src/app/state/account/types.ts

```diff
@@ -9,8 +9,14 @@ export interface BalanceDetails {
   total: StringifiedBigInt | null
 }
 
+export interface Allowance {
+  address: string
+  amount: string
+}
+
 export interface Account extends BalanceDetails {
   address: string
+  allowances?: Allowance[]
```

Comment:
We do not support monitor and ledger https://github.com/oasisprotocol/oasis-wallet-web/issues/791#issuecomment-1090487222

Reply:
Ah, it isn't used in oasismonitor.

Still:
- selectAccountAllowances shouldn't fall back to `[ ]` and lose the information that it's missing. It can do that in validateAllowance, since always setting new allowance is acceptable behaviour
- allowances is part of normal account information and not clearly reserved for paratimes, so it would be less of a pitfall if we can get it from oasismonitor or gRPC
- in which case it never needs to be undefined, and we don't need ` || [ ]`

Reply:
check moved to validateAllowance